### ISSUE TITLE
Log channel?

### ISSUE THREAD

**Issue by cyb3rgh05t** [Sleepingpirates/Invitarr#31]
will it be possible to readd a log channel in setup commands? like in the version before?
Thanks

**Comment by Sleepingpirates** [comment]
I will be adding this soon.

**Comment by jgog** [comment]
+1 having that was nice

**Comment by amokster88** [comment]
Yes,log channel will be appreciated! Everytime have to open docker logs
